s11/splat_arg.rb

```diff
@@ -0,0 +1,18 @@
+def sum(*num)
+  aux = 0
+  num.each {|n| aux += n}
+  aux
+end
+
+p sum(1,2,3,4,5,6,7,8,9)
```

Comment:
What happens if you don't pass any args to the function?

Reply:
Since the splat of the parameter in the method allows passing an indefinite number of arguments, it also supports 0 arguments, so when entering it would print the auxiliary variable by skipping the Each Block.